### ISSUE TITLE
Tiled view and chat aggregation

### ISSUE THREAD

**Issue by Gringeo80** [getstation/desktop-app#236]
Hello there
Is it possible to have tiled view of the single apps so that I can see for example whatsapp and telegram on one windows, withouth switching?
A
and is there any option to have message aggregation on one windows, so that I see all latest messages from all messengers in one window, despite of the original app?

Thank you very much, Igor

**Comment by viktor44** [comment]
> Is it possible to have tiled view of the single apps so that I can see for example whatsapp and telegram on one windows, withouth switching?

Try to detach both apps (Window -> Detach Current Application) and adjust window sizes

> is there any option to have message aggregation on one windows, so that I see all latest messages from all messengers in one window, despite of the original app?

No. I'm afraid it's out of scope of this application

**Comment by Gringeo80** [comment]
Thank you so much for your quick reply. I really appreciate you are taking time to listen to customer.

Yes I have seen the detach option, but tiling directly in app is super interesting, like on rambox.

I saw the feature of aggregation on Superchat or Userlike and I think the feature of aggregations messages in one windows would be fantastic. I understand the scope is not menat as, but this is the same issue all apps have, from franz to ferdi to rambox to getstation.

Is there any way this could be integrated?

**Comment by viktor44** [comment]
> I saw the feature of aggregation on Superchat or Userlike

Have you tried to add those applications as custom app to Station? Maybe that could be a solution you are looking for.
If they work properly we can add they to main app set

**Comment by Gringeo80** [comment]
Yes sure, but then I do not need Station. Superchat does already aggregate the messages, but we do not own it and it is quite expensive.
So I was wondering if you plan to do something like this, since this would help really to manage multi messages.

**Comment by Gringeo80** [comment]
Is there any way we could achieve this wish request? That would be very great
